www/nginx/src/opnsense/mvc/app/controllers/OPNsense/Nginx/forms/httpserver.xml

```diff
@@ -136,8 +136,16 @@
     <id>httpserver.ocsp_stapling</id>
     <label>OCSP Stapling</label>
     <type>checkbox</type>
-    <help>If you check this box, OCSP stapling (ssl_stapling) is enabled.</help>
-  </field>  
+    <advanced>true</advanced>
+    <help><![CDATA[If you check this box, OCSP stapling is enabled. If your Certificat has "TLS Feature: status_request"  (<a href="https://tools.ietf.org/html/rfc7633target="_blank">RFC 7633</a>) enabled, i.E. lets encrypt  "OCSP Must Staple", and don't enable OCSP Stapling you could get an error like "MOZILLA_PKIX_ERROR_REQUIRED_TLS_FEATURE_MISSING"]]></help>
```

Comment:
I'm not 100% pleased with the text, any suggestions?

Reply:
Maybe that is slightly better:
```suggestion
    <help><![CDATA[If you check this box, OCSP stapling is enabled. If your certificate has the TLS feature "status_request"  enabled and don't enable OCSP Stapling, you could get an TLS error like "MOZILLA_PKIX_ERROR_REQUIRED_TLS_FEATURE_MISSING". This can be the case if you use Let's Encrypt with "OCSP Must Staple". More information about OCSP stapling can be found <a href="https://tools.ietf.org/html/rfc7633" target="_blank">in RFC 7633</a>.]]></help>
```